### PR TITLE
modify sql script to correct value in column candidate_inactive

### PR DESCRIPTION
**Pull request by hcaofec**
sql script was modified to correct value in candidate_inactive column.
#2706

### CODE REVIEW

**Comment by hcaofec** [comment]
@LindsayYoung @ccostino Please review

File: data/sql_updates/candidate_history.sql [hunk 1]
```diff
@@ -78,7 +78,7 @@ left join elections using (cand_id)
 left join dates using (cand_id)
 left join disclosure.cand_inactive inactive on
     fec_yr.cand_id = inactive.cand_id and
-    fec_yr.fec_election_yr < inactive.election_yr
```
Comment (LindsayYoung) [inline data/sql_updates/candidate_history.sql]: I think it is supposed to be < because we want to join with all years before the candidate was inactive.

Comment (hcaofec) [reply data/sql_updates/candidate_history.sql]: Current logic is:      
" inactive.cand_id IS NOT NULL AS candidate_inactive 
  LEFT JOIN disclosure.cand_inactive inactive ON fec_yr.cand_id::text = inactive.cand_id::text AND fec_yr.fec_election_yr < inactive.election_yr  "
So when left joining disclosure.cand_inactive, it always returns an " inactive.cand_id " as long as the year is before the candidate was inactive.  And that puts "TRUE" to candidate_inactive column even though the candidate is actually active in that year.

Here is an example H4MT01041
@LindsayYoung

Comment (LindsayYoung) [reply data/sql_updates/candidate_history.sql]: Thanks for finding and fixing this

**Pull request merged**